src/commands/resume.rs

```diff
@@ -24,11 +24,11 @@ pub async fn resume(
     let queue = handler.queue();
 
     if queue.is_empty() {
-        return create_response(&ctx.http, interaction, QUEUE_IS_EMPTY).await;
+        return create_response(&ctx.http, interaction, NOTHING_IS_PLAYING).await;
     }
 
     if queue.resume().is_ok() {
-        return create_response(&ctx.http, interaction, "Resumed!").await;
+        return create_response(&ctx.http, interaction, "▶️  Resumed!").await;
```

Comment:
Spaces are being ignored?

![image](https://user-images.githubusercontent.com/5618493/150650791-20fcd6bf-4564-45a7-8fb4-8a0ccd459015.png)

Reply:
Wow, this was harder to fix than it should... It is Discord's fault, but eventually after some searching I found a fix on a [reddit comment](https://www.reddit.com/r/Discord_Bots/comments/qjageg/comment/hiouo52/?utm_source=reddit&utm_medium=web2x&context=3)!

Reply:
Nevermind, this is what happens on MacOS 
![image](https://user-images.githubusercontent.com/19473034/150657283-ea9adf8a-1d7b-43e1-a588-b63064f97081.png)

Reply:
Okay, back on track! This one (https://www.compart.com/en/unicode/U+2002) works on both platforms!
However, we'll need to remember that it is a different space on new strings!
![image](https://user-images.githubusercontent.com/19473034/150657379-11bdb65e-cbd5-4c08-915e-a19ff5397031.png)